docs/services/google_analytics_gtag.rst

```diff
@@ -34,10 +37,12 @@ template.  Insert the tag at the bottom of the HTML head::
     {% load google_analytics_gtag %}
     <html>
     <head>
-    {% google_analytics_gtag %}
     ...
     </head>
+    <body>
     ...
+    {% google_analytics_gtag %}
+    </body>
```

Comment:
Adding a `<script>` tag to the HTML `<head>` is illegal. So I guess the setup should be like this.

@mbourqui, @jcassee: Can you confirm?

Reply:
According to the [google doc](https://developers.google.com/analytics/devguides/collection/gtagjs#install_the_global_site_tag), it must be in that `<head>` tag:

> immediately after the \<head\> tag

Reply:
> Adding a <script> tag to the HTML <head> is illegal.

Are you sure? I think the `<script>` tag is metadata, and [`<head>` can contain metadata](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head)?

Reply:
The docs you quote clearly state:

> immediately _after_ the `<head>` tag

So, it has to go into the `<body>` tag, (ideally) as the very first data item.

Reply:
It literally means the opening `<head>` tag, as one can read it on [this help page](https://support.google.com/analytics/answer/1008080?hl=en&ref_topic=1008079#basic). There is some more explicit usage example on [that other help page](https://support.google.com/displayvideo/answer/7562276).

Reply:
Hmm, probably not clearly enough: The seem to say, "after the (opening) `<head>` tag". The [W3 validator service](https://validator.w3.org/nu/#textarea) proves this is allowed. :man_facepalming: :bomb: